project/RELEASE-VERSIONING.md

```diff
@@ -0,0 +1,28 @@
+# VIC Release Versioning
+
+This document provides examples of VIC release versions.
+
+The VIC project follows the general vX.Y.Z versioning scheme. Where X is Major, Y is
```

Comment:
Updated VERSION to reference semver.org. Merged files. We can split them later if it gets to large.